src/runtime/containerd-shim-v2/create.go

```diff
@@ -259,3 +272,98 @@ func doMount(mounts []*containerd_types.Mount, rootfs string) error {
 	}
 	return nil
 }
+
+func configureNonRootHypervisor(runtimeConfig *oci.RuntimeConfig) error {
+	userName, err := createVmmUser()
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err != nil {
+			removeVmmUser(userName)
```

Comment:
Shouldn't this be called when the shim ends too, to avoid leaving an unusable temporary user in the password file?

Reply:
The created user is deleted in the [cleanupVM()](https://github.com/fengwang666/kata-containers/blob/6ddee62128e10cef37259697224593b8cc53865a/src/runtime/virtcontainers/qemu.go#L1030-L1037) call. I didn't reuse the code because the logger is different.